glew.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: glew
+  version: 2.2.0
+  epoch: 0
+  description: "A cross-platform C/C++ extension loading library"
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - cmake
+      - glu
+      - libx11-dev
+      - libxi-dev
+      - libxmu-dev
+      - mesa-dev
+      - mesa-gl
+      - wolfi-base
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://downloads.sourceforge.net/glew/glew-${{package.version}}.tgz
+      expected-sha512: 57453646635609d54f62fb32a080b82b601fd471fcfd26e109f479b3fef6dfbc24b83f4ba62916d07d62cd06d1409ad7aa19bc1cd7cf3639c103c815b8be31d1
+
+  - runs: |
+      cd build
+      cmake -DBUILD_UTILS=ON -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_BUILD_TYPE=MinSizeRe ./cmake
```

Comment:
Can we use the cmake pipeline

Reply:
If I add
```yaml
 - working-directory: /home/build/build/cmake
     uses: cmake/configure
     with:
       opts: |
         -DBUILD_UTILS=ON 
```

the binaries arent generated and have sobpackage `mv` error
```bash
2024/05/03 21:40:21 WARN mv: can't rename '/home/build/melange-out/glew/usr/bin/glewinfo': No such file or directory
2024/05/03 21:40:21 ERRO Step failed: exit status 1
```

Reply:
what do you think?

Reply:
Let me get back to you on this I was able to build this locally with the cmake pipelines

Reply:
Thank you!